lib/strategy.js

```diff
@@ -78,7 +82,12 @@ Strategy.prototype.userProfile = function (accessToken, done) {
           { value: json.profile },
           { value: json.profile_medium },
         ];
-        profile.token = accessToken;
+        
+        if (json.sex === 'M') {
+          profile.gender = 'male';
+        } else if (json.sex === 'F') {
+          profile.gender = 'female';
+        }
```

Comment:
In my opinion this should be done by the person consuming this lib, we should only return them the data provided by Strava.